### PR TITLE
apply: update slow log place

### PR DESCRIPTION
**Pull request by siddontang**
Use slow log after writing a batch. 

PTAL @BusyJay @javaforfun @overvenus

### CODE REVIEW

File: src/raftstore/store/worker/apply.rs [hunk 1]
```diff
@@ -1449,14 +1442,16 @@ impl Runner {
             delegates: delegates,
             notifier: notifier,
             sync_log: sync_log,
+            tag: format!("[store {}]", store.store_id()),
```
Comment (overvenus) [inline src/raftstore/store/worker/apply.rs]: Could you aviod allocation here ?

Comment (siddontang) [reply src/raftstore/store/worker/apply.rs]: There is only one allocation, no need to optimize.

**Comment by javaforfun** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

**Pull request merged**